doc/lua_api.txt

```diff
@@ -6516,8 +6517,8 @@ object you are working with still exists.
     * `frame_speed` sets the animations frame speed. Default is 30.
 * `get_local_animation()`: returns idle, walk, dig, walk_while_dig tables and
   `frame_speed`.
-* `set_eye_offset(firstperson, thirdperson)`: defines offset vectors for camera
-  per player.
+* `set_eye_offset([firstperson, thirdperson])`: defines offset vectors for
```

Comment:
calling this with no arguments doesn't make sense to me. I don't think these are even the correct defaults?

Reply:
This makes sense in order to reset eye offset for example after detaching from an entity. The default values are correct.

Reply:
I would have thought that they would have the same zero position, meaning that setting the third person offset to 0,0,0 would disable it?